packages/cadl-ranch-coverage-sdk/src/constants.ts

```diff
@@ -1 +1,22 @@
-export const GeneratorMode: string[] = ["azure", "standard"];
+export enum GeneratorMode {
+  azure = "azure",
+  standard = "standard",
+}
+export const TableConfigs: TableConfig[] = [
```

Comment:
this feels like this should not be part of the SDK no?

Reply:
Right, it should be part of the dashboard. Fixed.